packages/private-build-infra/src/addon-build-config-for-data-package.js

```diff
@@ -45,35 +45,28 @@ function addonBuildConfigForDataPackage(PackageName) {
       }
     },
 
-    _suppressUneededRollupWarnings(message, next) {
+    _handleRollupWarning(message) {
       if (message.code === 'CIRCULAR_DEPENDENCY') {
         return;
-      } else if (message.code === 'NON_EXISTENT_EXPORT') {
+      } else if (message.code === 'NON_EXISTENT_EXPORT' && message.message.includes('ts-interfaces')) {
```

Comment:
the removed slashes were there intentionally although I do see how it might look like a borked regex ;)

It's unlikely for this shorter string to match anything the longer string didn't match but the slashes were insurance we were finding a warning.

On the code style change I'm mixed. Folks rushed into using `contains` and `includes` and `startsWith` in our library and test code and I recently had to revert all of that to get us back to IE11 support. I like using new things but I dislike mixing styles. Overall though this is clearly a build file so its a bit more clear that it works here.

Reply:
> Folks rushed into using contains and includes and startsWith in our library and test code and I recently had to revert all of that to get us back to IE11 support. I like using new things but I dislike mixing styles.

Ya, but this is Node code and has no requirement for IE11 support.